### PR TITLE
Update facts.py

### PR DESCRIPTION
**Pull request by Deijdane**

### CODE REVIEW

File: class01-git/facts.py [hunk 1]
```diff
@@ -11,6 +11,9 @@
            facts=['I can meow like a cat', 'I am a PhD student', 'I am not Estonian']),
     Person(github_login="cat", first_name="Martha", last_name="KittyCat",
            facts=['I am a dog',"I can say meow", "I like milk"]),
+    Person(github_login="Deijdane", first_name="Aubin", last_name="Medjaed",
+           facts=['Cats eat grass.',"I don't like grass.", "I'm not a cat person, more a dog person."]),
```
Comment (Javanochka) [inline class01-git/facts.py:15]: Great that you added facts about yourself. Could you please also add the number of the right one to a corresponding dictionary in the code?

File: class01-git/facts.py [hunk 2]
```diff
@@ -11,6 +11,9 @@
...
+    Person(github_login="Hawawou", first_name="Hawawou", facts=['I am a student', 'I am from Togo', 'I love cats'])
 ]
 #facts are numerated from 0
 answers = {'example': 1,
```
Comment (Javanochka) [inline class01-git/facts.py:19]: Here!

**Review by Javanochka — CHANGES_REQUESTED** [review]
I have some small comments. Make sure to change your code and inform me :)

File: class01-git/facts.py (hunk 2 above)
Comment (Deijdane) [reply class01-git/facts.py:19]: Should be resolved

**Comment by Javanochka** [comment]
Good work!

**Pull request closed without merging**